### DISCUSSION TITLE
Shift behaviour in drag and drop from file explorer

### DISCUSSION THREAD

**Discussion by MaxFSP** [microsoft/vscode-discussions#1621] [Extension Development QnA] [answered]
I was wondering if its possible to re wrtite how the drag and drop thing works, im doing a chat extension thing... and to drag and drop images onto it i need to hold shift from the moment i click on the image to succesfully have it in the chat (this is already working) and i was wondering if i had to re write maybe something on the editor itself or if i can just modify that behavior to not require the hold of the shift key at all...

**Comment by isidorn** [comment, marked as answer]
Currently your extension can not control this. But we are working on a better drag and drop behaviour in Chat
Would be curious to learn what kind of chat extension you are doing.

**Reply by MaxFSP** [reply]
Interesting, i am also working with the vscodium repo, is there a way to modify the shift behavior for my extension view from there?
Thanks for answering!

**Reply by isidorn** [reply]
I do not think so. But I do not work on vscodium.
